Sources/PortalSwift/Components/PortalWebView.swift

```diff
@@ -463,7 +459,23 @@ public class PortalWebView: UIViewController, WKNavigationDelegate, WKScriptMess
     }
   }
 
-  private func injectPortal(address: String, apiKey: String, chainId: String, gatewayConfig: String, autoApprove _: Bool = false, enableMpc: Bool = false) -> String {
-    "window.portalAddress = \"\(address)\";window.portalApiKey = \"\(apiKey)\";window.portalAutoApprove = true;window.portalChainId = \"\(chainId)\";window.portalGatewayConfig = \"\(gatewayConfig)\";window.portalMPCEnabled = \"\(String(enableMpc))\";\(PortalInjectionScript.SCRIPT as Any)true;"
+  private func injectPortal(address: String, apiKey _: String, chainId: String, gatewayConfig: String, autoApprove _: Bool = false, enableMpc: Bool = false) -> String {
+    "window.portalAddress = \"\(address)\";window.portalAutoApprove = true;window.portalChainId = \"\(chainId)\";window.portalGatewayConfig = \"\(gatewayConfig)\";window.portalMPCEnabled = \"\(String(enableMpc))\";\(PortalInjectionScript.SCRIPT as Any)true;"
```

Comment:
shouldn't autoapprove be a variable here instead of hard coded to true

Reply:
No. Because the injected provider isn't responsible for handling the approval process, the SDK is. So this allows everything to get passed back up by the signer without an approval hook in the DOM itself. This is how it's always worked.